.github/workflows/CICD.yml

```diff
@@ -83,12 +83,11 @@ jobs:
       fail-fast: false
       matrix:
         job:
-          # { os, target, cargo-options, features, use-cross }
           - { os: ubuntu-latest  , target: arm-unknown-linux-gnueabihf , use-cross: true }
           - { os: ubuntu-18.04   , target: aarch64-unknown-linux-gnu   , use-cross: true }
           - { os: ubuntu-18.04   , target: i686-unknown-linux-gnu      , use-cross: true }
           - { os: ubuntu-18.04   , target: i686-unknown-linux-musl     , use-cross: true }
-          - { os: ubuntu-18.04   , target: x86_64-unknown-linux-gnu    }
+          - { os: ubuntu-18.04   , target: x86_64-unknown-linux-gnu                      , disable-deploy: true }  ## deployed from ubuntu-16.04
```

Comment:
The change looks great. But it makes me wonder. Why do we even run the exact same job on Ubuntu 18.04 and 16.04? It's not really our job to make sure that everything still works on 16.04. The Rust toolchain should be exactly the same (?) so I'm not sure if there's any benefit in the 16.04 job.

What do you think?

Reply:
Considering that 16.04 reaches end-of-life in 3 months, I agree it makes sense to stop building for it and just use 18.04 instead. I'll update this PR.